webapp/views/App/views/Analysis/Chains/HeaderLeft/HeaderLeft.js

```diff
@@ -1,17 +1,17 @@
 import React from 'react'
-import { useDispatch } from 'react-redux'
 import { useHistory } from 'react-router'
 
 import { useI18n } from '@webapp/store/system'
-import { createChain } from '@webapp/loggedin/modules/analysis/chains/actions'
+import { analysisModules, appModuleUri } from '@webapp/app/appModules'
 
 const HeaderLeft = () => {
-  const dispatch = useDispatch()
   const history = useHistory()
   const i18n = useI18n()
 
+  const onClickNew = () => history.push(`${appModuleUri(analysisModules.processingChain)}`)
+
   return (
-    <button type="button" onClick={() => dispatch(createChain(history))} className="btn btn-s">
+    <button type="button" onClick={onClickNew} className="btn btn-s">
```

Comment:
you can use ```<Link to={appModuleUri(analysisModules.processingChain)} className='btn btn-s' />```

Reply:
Yes! Now we don’t need the action because we don’t do anything else